pyproject.toml

```diff
@@ -11,8 +11,8 @@ requires = [
     "requests"
 ]
 description-file = "README.md"
-requires-python = ">=3.4"
-
+requires-python = ">=3.5"
+\be
```

Comment:
Looks like this might have snuck in by vim accident?

Reply:
Heh, yes I didn't notice this!